src/vs/workbench/contrib/terminal/browser/terminalStatusList.ts

```diff
@@ -22,32 +22,10 @@ export const enum TerminalStatus {
 	Bell = 'bell',
```

Comment:
Curious, when is this shown and how does it look?

Reply:
In bash for example when you hit left in an empty prompt you see this for a second:

![image](https://user-images.githubusercontent.com/2193314/225419966-f4c3f042-54cc-4684-93d5-d0a337fd74d4.png)